test/data/marriage-abroad-files.yml

```diff
@@ -370,8 +372,8 @@ lib/smart_answer_flows/marriage-abroad/questions/partner_opposite_or_same_sex.go
 lib/smart_answer_flows/marriage-abroad/questions/what_is_your_partners_nationality.govspeak.erb: 80e04f36c75c232bede1a244d621471e
 lib/data/rates/marriage_abroad_consular_fees.yml: 20afc9d15de06e0a199761433b055d9f
 lib/data/marriage_abroad_services.yml: a487a1dfe5db504bc0fb6c72b8a099c1
-lib/smart_answer/calculators/marriage_abroad_calculator.rb: c809619bc6139214429b60b99fc206b4
+lib/smart_answer/calculators/marriage_abroad_calculator.rb: 32b5b32dc3e56cd41e9a881db6bf4dd7
 lib/smart_answer_flows/shared/_overseas_passports_embassies.govspeak.erb: 0bc67c4469fe692a4fa53c9b461821bf
-lib/smart_answer/calculators/marriage_abroad_data_query.rb: e6aff205028c84b369cf9faefb3a51a6
+lib/smart_answer/calculators/marriage_abroad_data_query.rb: 38fee4a2ef31df00feb780397d676237
```

Comment:
I noticed this line.. there isn't any chang here for `lib/smart_answer/calculators/marriage_abroad_data_query.rb`
The only change thus far for the aforementioned can be found in 5b0e06a.. Probably best to move this to that commit or have a separate checksum commit..